projects/chaos/src/emergency_fault.c

```diff
@@ -0,0 +1,77 @@
+#include "emergency_fault.h"
+
+#include <stddef.h>
+#include <stdint.h>
+
+#include "can_ack.h"
+#include "can_msg_defs.h"
+#include "can_transmit.h"
+#include "chaos_events.h"
+#include "critical_section.h"
+#include "soft_timer.h"
+#include "status.h"
+
+#define EMERGENCY_FAULT_BACKOFF_MS 10
```

Comment:
We can probably make this in the 100s of milliseconds

Reply:
Done.